addons/website_form/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -9,11 +9,8 @@
             overflow-y: auto;
 
             table {
-                table-layout: fixed;
-                width: 100%;
-
                 input {
-                    width: 100%;
+                    width: 192px;
```

Comment:
Ok your solution works but I don't really understand why...
Indeed the wrong width of we-list elements comes when a `<tr>` has a position absolute. By looking the debug console it looks like there is a strange margin on the first and last `<we-button>` of trs. Isn't the fix about preventing those margins to appear ?
https://youtu.be/4LBaXLHC4ww

Reply:
@Guillaume-gdi When the first `<tr>` element is given a position `absolute` (while being dragged), because the table is set to `"table-layout: fixed;"` the other `<tr>` elements lose their reference to the size of the first `<tr>` element. As a result, they automatically resize themselves by dividing the available space proportionally (in this case, one third of the space each since there are three elements) and thus create these margins.

Reply:
@Guillaume-gdi The code you propose changes the style of the table. With this the first and last `<td>` lose their 28px width. Actually, 192px is not random, this is the width of the input before my change. Sure it would have been better to use a percentage or a variable but that's the only solution I found. And since the panel is not responsive, I considered that it was not a problem.

Reply:
I really prefer if there is no 192px value... :grimacing: 
Can't you solve the size of the first and last `<td>` by setting the width of the first and last we-button to 28px too ? :thinking: